orion/controllers/announce/announceController.go

```diff
@@ -33,7 +34,7 @@ func GetAnnouncement(c *gin.Context) {
 	if _, err := GetAnnouncementById(id); err != nil {
 		panic(err)
 	} else {
-		c.Status(http.StatusOK)
+		c.Status(http.StatusNoContent)
```

Comment:
uh oh, I think this is a mistake. This function is supposed to return one singular Announcement object.
So, this should be `c.Json(http.StatusOK, announceObj)`

Reply:
I should've caught this. Argh I can't wait until I finish setting up a better unit test framework.

Reply:
Hmm I think think it used to say c.Json, but then you told me to change it to c.Status.

Reply:
ah I see. So if we are in the situation where we return StatusOK and no other content, we can simply just use Status(NoContent).
On the other hand, if we are returning additional values (in this case a program object), we should use c.Status(OK, object).